phoenixRest/views/user/instance/__init__.py

```diff
@@ -48,11 +49,14 @@
 class UserInstanceResource(object):
     def __acl__(self):
         acl = [
-            # Who has access to view an user?
+            # Who has access to view a user?
             (Allow, HR_ADMIN, 'user_view'),
             (Allow, ADMIN, 'user_view'),
             (Allow, TICKET_ADMIN, 'user_view'),
             (Allow, TICKET_CHECKIN, 'user_view'),
+            # Who has acces to user friendship status?
+            (Allow, HR_ADMIN, 'get_friendship_states'),
+            (Allow, ADMIN, 'get_friendship_states'),
```

Comment:
I was going to ask "why get_friendship_states" but since the list doesn't represent friendships its probably a good idea.